tests/tests-core/src/main/java/com/gentics/mesh/core/schema/MicroschemaEndpointTest.java

```diff
@@ -318,7 +318,7 @@ public void testUpdateWithBogusUuid() throws GenericRestException, Exception {
 
 			call(() -> client().updateMicroschema("bogus", request), NOT_FOUND, "object_not_found_for_uuid", "bogus");
 
-			Microschema reloaded = boot().microschemaContainerRoot().findByUuid(microschema.getUuid());
+			Microschema reloaded = boot().meshRoot().getMicroschemaContainerRoot().findByUuid(microschema.getUuid());
```

Comment:
Can be rewritten like
HibMicroschema reloaded = boot().microschemaDao().findByUuidGlobal(microschema.getUuid());
(all occurrances)

Reply:
IIRC I did this in the later pull requests... let me check.

Reply:
I did it for limited number of cases, to avoid pull requests overload. Let's see how much will it take to replace all the common code occurrences...

Reply:
Replaced where possible. The missing test cases will be anyway reviewed during the Enterprise project development.